public/js/interface.js

```diff
@@ -0,0 +1,50 @@
+$(document).ready(function() {
+	var game = new Game();
+
+	$('#zero').click(function() {
+		game.rollBall(0);
+	});
+
+	$('#one').click(function() {
+		game.rollBall(1);
+	});
+
+	$('#two').click(function() {
+		game.rollBall(2);
+	});
+
+	$('#three').click(function() {
+		game.rollBall(3);
+	});
+
+	$('#four').click(function() {
+		game.rollBall(4);
+	});
+
+	$('#five').click(function() {
+		game.rollBall(5);
+	});
+
+	$('#six').click(function() {
+		game.rollBall(6);
+	});
+
+  $('#seven').click(function() {
+		game.rollBall(7);
+	});
+
+	$('#eigth').click(function() {
+		game.rollBall(8);
+	});
+
+	$('#nine').click(function() {
+		game.rollBall(9);
+	});
+
+	$('#ten').click(function() {
+		game.rollBall(10);
+	});
+
+	$('#frame1').text(game.rollsArray[0,1]);
```

Comment:
Expected ']' to match '[' from line 48 and instead saw ','.<br>Expected ')' and instead saw '1'.<br>Expected an identifier and instead saw ']'.<br>Expected an assignment or function call and instead saw an expression.<br>Expected an identifier and instead saw ')'.